app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -1088,17 +1424,244 @@
+				}
+			},
+			{
+				"item": "viz_9VBWSTo0",
+				"type": "block",
+				"position": {
+					"x": 840,
+					"y": 40,
 					"w": 90,
 					"h": 40
 				}
+			},
+			{
+				"item": "viz_W3FbUfC5",
+				"type": "block",
+				"position": {
+					"x": 840,
+					"y": 90,
+					"w": 90,
+					"h": 40
+				}
+			},
+			{
+				"item": "viz_ITQi01CO",
+				"type": "block",
+				"position": {
+					"x": 660,
+					"y": 80,
+					"w": 180,
+					"h": 40
+				}
+			},
+			{
+				"item": "viz_u7N1Tqw0",
+				"type": "block",
+				"position": {
+					"x": 650,
+					"y": 150,
+					"w": 540,
+					"h": 190
+				}
+			},
+			{
+				"item": "viz_9w03Dvsi",
+				"type": "block",
+				"position": {
+					"x": 660,
+					"y": 144,
+					"w": 130,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_h43wWbvn",
+				"type": "block",
+				"position": {
+					"x": 890,
+					"y": 180,
+					"w": 100,
+					"h": 20
+				}
+			},
+			{
+				"item": "viz_COobqNkb",
+				"type": "block",
+				"position": {
+					"x": 1050,
+					"y": 160,
+					"w": 70,
+					"h": 20
+				}
+			},
+			{
+				"item": "viz_NKJrrFw7",
+				"type": "block",
+				"position": {
+					"x": 1050,
+					"y": 180,
+					"w": 100,
+					"h": 20
+				}
+			},
+			{
+				"item": "viz_BkpIipBR",
+				"type": "block",
+				"position": {
+					"x": 890,
+					"y": 160,
+					"w": 90,
+					"h": 20
+				}
+			},
+			{
+				"item": "viz_RDVKOOtq",
+				"type": "line",
+				"position": {
+					"from": {
+						"x": 665,
+						"y": 217
+					},
+					"to": {
+						"x": 1170,
+						"y": 219
+					}
+				}
+			},
+			{
+				"item": "viz_BUKru8jJ",
+				"type": "block",
+				"position": {
+					"x": 860,
+					"y": 220,
+					"w": 150,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_XaUYOAz0",
+				"type": "block",
+				"position": {
+					"x": 660,
+					"y": 230,
+					"w": 200,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_VlzsP6MX",
+				"type": "block",
+				"position": {
+					"x": 1020,
+					"y": 220,
+					"w": 150,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_ckkvOrLk",
+				"type": "block",
+				"position": {
+					"x": 660,
+					"y": 270,
+					"w": 240,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_ELaJTTU7",
+				"type": "block",
+				"position": {
+					"x": 860,
+					"y": 270,
+					"w": 150,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_oBFyE8G8",
+				"type": "block",
+				"position": {
+					"x": 1020,
+					"y": 270,
+					"w": 150,
+					"h": 50
+				}
+			},
+			{
+				"item": "viz_lQdKYbSx",
+				"type": "line",
+				"position": {
+					"from": {
+						"x": 662,
+						"y": 272
+					},
+					"to": {
+						"x": 1167,
+						"y": 274
+					}
+				}
+			},
+			{
+				"item": "viz_69L4jcHH",
+				"type": "block",
+				"position": {
+					"x": 660,
+					"y": 306,
+					"w": 190,
+					"h": 30
+				}
 			}
-		],
-		"type": "absolute"
+		]
 	},
 	"title": "Dashboard",
 	"description": "Represents data received from Red Hat Insights"
-}]]>
+}}]]>
```

Comment:
I'm getting "Layout undefined is not defined", can it be because of this change?

![Screenshot 2022-08-23 at 18-15-24 Dashboard](https://user-images.githubusercontent.com/7695766/186209384-6993b6cb-c64d-4049-94c4-ee82521e47ef.png)

Reply:
fixed here: 8831984ad74d1b3b17863094c46fb22bab94e478